bin/yaml/libraries.yaml

```diff
@@ -1025,7 +1025,7 @@ libraries:
         targets:
         - trunk
         type: github
-      beman_iterator:
+      beman_iterator_interface:
```

Comment:
Is it safe to rename these keys? Alternatively does this require an update in the main repo too? cc @partouf

Reply:
Judging from https://github.com/compiler-explorer/compiler-explorer/pull/7084/files, I guess this library was never added to live? Do you happen to know @changkhothuychung?

Reply:
not super happy with this as it changes the directory name where it's stored as well and will take some manual effort to make correct, but can deal with it

Reply:
1 yes
2 the name "must" (technically not really in this case, but in general it must) be the same as the id used in the CE repo and there you can't use `.`, so you cannot use it here either
3 yes

Reply:
Cool. Then I'll add am internal Beman recommendation similar to: `Any beman.<short_name> Beman library should be converted to `beman_<short_name>` for filling in the keys in YAML/configs inside the Compiler Explorer infrastructure. The display name can be still set to beman.<short_name>, via `libs.beman_<shortname>.name = beman.<short_name>.`